upup/pkg/kutil/rollingupdate_cluster_validate_drain.go

```diff
@@ -0,0 +1,313 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
The logic here is not that different to normal rollingupdate.  I feel like it boils down to whether we run validate, and whether we call drain, which could just be flags?  I don't think we need to copy-pasta this.